closed/get_openssl_source.sh

```diff
@@ -23,16 +23,17 @@
 # ===========================================================================
 
 usage() {
-	echo "Usage: $0 [-h|--help] [--openssl-version=<openssl version 1.0.2 and above to download>]"
+	echo "Usage: $0 [-h|--help] [--openssl-repo=<repo URL>] [--openssl-version=<openssl version 1.0.2 and above to download>]"
 	echo "where:"
 	echo "  -h|--help             print this help, then exit"
+	echo "  --openssl-repo        OpenSSL repo. By default, https://github.com/openssl/openssl.git"
```

Comment:
This should use the full word "repository" as well.

Reply:
Updated.